.github/workflows/deploy.yml

```
@@ -0,0 +1,26 @@
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.SERVER_IP }}
+          username: ${{ secrets.SERVER_USER }}
+          password: ${{ secrets.SERVER_PASSWORD }}
+          script: |
+            cd ${{ secrets.APP_PATH }}
+            git pull origin main
+            composer install --no-dev
+            php artisan migrate --force
+            php artisan optimize:clear
+            php artisan queue:restart
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)